synapseclient/client.py

```diff
@@ -821,7 +821,7 @@ def login(
 
         # Check version before logging in
         if not self.skip_checks:
-            version_check()
+            version_check(self.logger)
```

Comment:
What is wrong with the way you're passing in this variable? Is it being passed into logger?

Reply:
fixed